web/src/lib/components/asset-viewer/immich-thumbnail.svelte

```diff
@@ -29,33 +29,32 @@
 
 	const loadImageData = async () => {
 		if ($session.user) {
```

Comment:
I couldn't find a clean way to do this actually. Since a lot of pages have different behavior based on whether the user is found or not so I cannot put it into the `__layout.svelte`